src/ucp/core/ucp_ep.c

```diff
@@ -2698,9 +2698,8 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
         rma_config->max_put_bcopy    = SIZE_MAX;
         rma_config->max_get_bcopy    = SIZE_MAX;
 
-        if (ucp_ep_config_get_multi_lane_prio(
-                    config->key.rma_lanes, lane,
-                    ucs_static_array_size(config->key.rma_lanes)) == -1) {
+        if (ucp_ep_config_get_multi_lane_prio(config->key.rma_lanes, lane) ==
```

Comment:
line seems too long
pls check clang-format

Reply:
line is finished in column 78